.github/workflows/build-push-template.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and Push Playground Image
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-push-template:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build and push Docker image for Substrate Playground
+      uses: docker/build-push-action@v1 # https://github.com/docker/build-push-action
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        dockerfile: .devcontainer/Dockerfile
+        repository: jeluard/substrate-playground-template-recipes
+        tags: latest
+        add_git_labels: true
+        tag_with_ref: true
+        tag_with_sha: true
+
+    # https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter
+    - run: echo ::set-output name=image::jeluard/substrate-playground-template-recipes:sha-${GITHUB_SHA::7}
+      id: env
+
+    - name: Update devcontainer.json
+      run: |-
+        t=$(mktemp)
+        cat .devcontainer/devcontainer.json | jq ".image = \"${{ steps.env.outputs.image }}\"" > ${t} && mv ${t} .devcontainer/devcontainer.json
```

Comment:
Didn't you want to remove that?

Reply:
Oh right. I removed the part that makes the actual commit, but left this in by accident.